runtime/v1/shim/service.go

```diff
@@ -551,9 +551,11 @@ func shouldKillAllOnExit(bundlePath string) (bool, error) {
 	var bundleSpec specs.Spec
 	bundleConfigContents, err := ioutil.ReadFile(filepath.Join(bundlePath, "config.json"))
 	if err != nil {
-		return false, err
+		return true, err
+	}
+	if err := json.Unmarshal(bundleConfigContents, &bundleSpec); err != nil {
+		return true, fmt.Errorf("encountered error during Unmarshal, ignoring : %v", err)
```

Comment:
This is somewhat against Go's practices.
If we return an error, then we should not trust to the boolean value we return.
Should we just return bool and log errors, instead of returning? Or or least document unusual behavior of this func.
@fuweid WDYT?

Reply:
The caller of shouldKillAllOnExit just logs the returned error.
Let me drop the error return and move the log inside shouldKillAllOnExit.

Reply:
@mxpv Yeah. I think we don't need to return error here. Just log what happens and return bool.